.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -568,7 +570,7 @@ Device Trust requires Teleport Enterprise.
 
 This feature has additional build requirements, so it should be tested with a
 pre-release build from Drone (eg:
-`https://get.gravitational.com/teleport-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
+`https://get.gravitational.com/teleport-ent-v10.0.0-alpha.2-linux-amd64-bin.tar.gz`).
```

Comment:
Similarly here, but we actually want the Enterprise build in this case.

Reply:
update reference version here as well?